### ISSUE TITLE
Reader-issue with base-class names

### ISSUE THREAD

**Issue by kiselgra** [kiselgra/c-mera#40]
It seems that for base-class names C-Mera does not apply identifier conversion, that is

```
(class foo-bar ())
(class bar-foo ((public foo-bar)))
```

turns into

```
class foo_bar;

class bar_foo: public foo-bar;
```

Also, when only emitting forward-declarations base-class specifications should also be dropped.

**Comment by kiselgra** [comment]
A similar thing happens for ```decl```:
```
(decl ((long-type-name var-name)))
```
yields
```
long-type-name var_name;
```

This is especially conflicting with the way class-names are rewritten into C form as shown above, thus
```
(class foo-bar ...)
(decl ((foo-bar fb)) ...)
```
will not work.

**Comment by lispbub** [comment]
Solved in c-mera 1.0
